frontend/src/shared/components/nav/Nav.tsx

```diff
@@ -0,0 +1,18 @@
+import { Flex } from '@chakra-ui/react';
+import { NavLink } from './NavLink';
+
+export const Nav = () => {
+  return (
+    <Flex as='nav'>
+      <NavLink label='Home' route='/' />
+      <NavLink label='Accounts' route='/accounts' />
+      <NavLink label='Events' route='/events' isDisabled />
+      <NavLink label='Venues' route='/venues' isDisabled />
+      <NavLink label='Departments' route='/departments' isDisabled />
+      <NavLink label='Equipment' route='/equipment' isDisabled />
+      <NavLink label='Information' route='/information' isDisabled />
```

Comment:
when i click the disabled routes, they still navigate to the page and show a 404 error page. no need to change as it doesn't really matter but just something i noticed